app/Http/Controllers/Api/ApplicationsController.php

```
@@ -2876,8 +2876,12 @@ public function update_env_by_uuid(Request $request)
 
         $this->authorize('manageEnvironment', $application);
 
+        if ($request->has('key')) {
+            $request->merge(['key' => ValidationPatterns::normalizeEnvironmentVariableKey((string) $request->key)]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ValidationPatterns' in method 'update_env_by_uuid'. (undefined)

(StaticAccess)